src/styles/blocks/stars.css

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(./../../images/star.svg);
+  background-size: 80%;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(./../../images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(./../../images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(./../../images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(./../../images/star-active.svg);
+}
+
+.stars--5 :nth-child(n + 1) {
+  background-image: url(./../../images/star-active.svg);
+}
```

Comment:
These selectors must be separated by commas

Reply:
I fixed it